routes/apis.js

```diff
@@ -373,7 +374,9 @@ apis.getConfig = function (app, res, loggedInUserId, apiId) {
             if (!isAdmin) {
                 configReturn = {
                     api: {
-                        uris: configJson.api.uris
+                        uris: configJson.api.uris,
+                        host: configJson.api.host,
+                        protocol: url.parse(configJson.api.upstream_url).protocol 
```

Comment:
This code is only run through for non-admin users; will the protocol also be set by default for admin users? I don't think so, right?